velox/exec/SetAccumulator.h

```diff
@@ -106,10 +153,50 @@ struct SetAccumulator {
                                  : uniqueValues.size();
   }
 
+  void serializeNullIndex(char* buffer) {
```

Comment:
inconsistent naming: readNullIndex vs. serializeNullIndex

perhaps, use serialize and deserialize or (serialize + read) and (deserialize + write)